src/blocksmith/components/article-block-preview/index.js

```diff
@@ -36,7 +42,7 @@ function ArticleBlockPreview(props) {
           <tr>
             <th colSpan={2} className="nowrap ps-2 fs-5" scope="row">
               <Badge color="dark" className={`rounded-pill status-${status}`}>{status}</Badge>
-              <a href={url} className="ms-2" target="_blank">
+              <a href={externalUrl} className="ms-2" target="_blank">
```

Comment:
@mtakla16 In the event there is no externalUrl (null), then we should not render this.